r_markdown/emissions.Rmd

```diff
@@ -144,32 +126,174 @@ output:
+  save(fuel_burnt, file = "../r_files/fuel_burnt.Rda")
 ```
 
-### Merge fuel burnt with fuel properties
+___
+
+# Energy
+
+* timestamps
 
 ```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
-  fuel_burnt <- dplyr::left_join(fuel_burnt, dplyr::select(samples, id, fuel), by = "id") %>%
-	              dplyr::left_join(fuel_properties, by = "fuel")
+  e_times <- dplyr::filter(test_times, grepl("^start_[0-9]|^end_[0-9]", var)) %>%
+             dplyr::select(-date) %>%
+             tidyr::separate(var, c("var", "rep")) %>%
+             tidyr::spread(var, value)
 ```
 
-### Save
+* which pot
 
 ```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
-  save(fuel_burnt, file = "../r_files/fuel_burnt.Rda")
+  e_pot <- dplyr::select(wood_pot, -date) %>%
+           dplyr::bind_rows(dplyr::select(batch_pot, -date))
 ```
 
-___
+* pot mass
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  e_pot_mass <- dplyr::filter(wood_wgts,
+                              grepl("^pot",
+                              var)) %>%
+                tidyr::separate(var, c("var", "pot")) %>%
+                dplyr::select(-var, -date) %>%
+                tidyr::spread(pot, value)
+
+  e_pot_mass <- dplyr::bind_rows(e_pot_mass,
+                                 dplyr::bind_rows(e_pot_mass,
+                                 dplyr::filter(batch_wgts,
+                                               grepl("^pot",
+                                               var)) %>%
+                                 tidyr::separate(var, c("var", "pot")) %>%
+                                 dplyr::select(-var, -date) %>%
+                                 tidyr::spread(pot, value)))
+
+```
+
+* pot + water mass
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  e_pot_water_mass <- dplyr::filter(wood_wgts,
+                                    grepl("^on_[0-9]|^off_[0-9]", var)) %>%
+                      tidyr::separate(var, c("var", "rep")) %>%
+                      tidyr::spread(var, value) %>%
+                      dplyr::select(-date) %>%
+                      dplyr::rename(mass_on = on, mass_off = off)
+
+  e_pot_water_mass <- dplyr::bind_rows(e_pot_water_mass,
+                                       dplyr::filter(batch_wgts,
+                                       grepl("^on_[0-9]|^off_[0-9]", var)) %>%
+                                       tidyr::separate(var, c("var", "rep")) %>%
+                                       tidyr::spread(var, value) %>%
+                                       dplyr::select(-date) %>%
+                                       dplyr::rename(mass_on = on, mass_off = off))
+```
+
+* extract temperature for periods of interest
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  e_temp <-  dplyr::select(temp, id, datetime, time, tamb, thood)
```

Comment:
did variable names get updated in temp.Rda file but not here?? I see "t_pot" and "t_hood"